.gitattributes

```diff
@@ -0,0 +1,4 @@
+* text=auto
+
+*.sh           text eol=lf
+*.bat          text eol=crlf
```

Comment:
These two lines are technically not necessary in this project.  
However, I'd like to always add them nonetheless, as it prevents issues on WIndows machines with any possible future scripts.

Reply:
Seems reasonable